### ISSUE TITLE
beets: replaygain gstreamer setup broken on unstable and master

### ISSUE THREAD

**Issue by pimeys** [NixOS/nixpkgs#184175]
### Describe the bug

Using the beets package together with replaygain setting stopped working on unstable and master branches.

### Steps To Reproduce

With the following config:

https://github.com/pimeys/nixos/blob/defa1c79d9c1e917eb864f26e00ecadf568f8ca9/desktop/beets/default.nix#L8

Beets will not launch and just returns an error:

```console
❯ beet
error: replaygain initialization failed: Failed to load GStreamer 1.0: Namespace Gst not available
```

### Expected behavior

Beets should launch correctly without a error.

### Additional context

A workaround to make it work again is to replace the `gstreamer` backend to `command`.

### Notify maintainers

@aszlig @doronbehar @lovesegfault @pjones

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
❯ nix-shell -p nix-info --run "nix-info -m"
this path will be fetched (0.00 MiB download, 0.00 MiB unpacked):
  /nix/store/q124axggd6wl6fh9508sxgkknkifrl0z-nix-info
copying path '/nix/store/q124axggd6wl6fh9508sxgkknkifrl0z-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.18.12, NixOS, 22.11 (Raccoon)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.3`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by MartinPotier** [comment]
This also breaks the mopidy-youtube extension, with the same error:

```
Aug 02 08:39:44 wrk mopidy[1430]:     from mopidy import commands
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/qdky761j3im8sn13rmpj11x4hy84rkqa-mopidy-with-extensions-3.3.0/lib/pyth>
Aug 02 08:39:44 wrk mopidy[1430]:     from mopidy.audio import Audio
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/qdky761j3im8sn13rmpj11x4hy84rkqa-mopidy-with-extensions-3.3.0/lib/pyth>
Aug 02 08:39:44 wrk mopidy[1430]:     from .actor import Audio
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/qdky761j3im8sn13rmpj11x4hy84rkqa-mopidy-with-extensions-3.3.0/lib/pyth>
Aug 02 08:39:44 wrk mopidy[1430]:     from mopidy.audio import tags as tags_lib
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/qdky761j3im8sn13rmpj11x4hy84rkqa-mopidy-with-extensions-3.3.0/lib/pyth>
Aug 02 08:39:44 wrk mopidy[1430]:     from mopidy.internal.gi import GLib, Gst
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/qdky761j3im8sn13rmpj11x4hy84rkqa-mopidy-with-extensions-3.3.0/lib/pyth>
Aug 02 08:39:44 wrk mopidy[1430]:     gi.require_version("Gst", "1.0")
Aug 02 08:39:44 wrk mopidy[1430]:   File "/nix/store/d0m1mk1zbsjysrm1bq44087njrygnl7w-python3.10-pygobject-3.42.1/lib/pytho>
Aug 02 08:39:44 wrk mopidy[1430]:     raise ValueError('Namespace %s not available' % namespace)
Aug 02 08:39:44 wrk mopidy[1430]: ValueError: Namespace Gst not available
Aug 02 08:39:44 wrk systemd[1]: mopidy.service: Main process exited, code=exited, status=1/FAILURE
Aug 02 08:39:44 wrk systemd[1]: mopidy.service: Failed with result 'exit-code'.
```

**Comment by lilyinstarlight** [comment]
Mopidy should be fixed by #184015 in the next nixos-unstable bump. I imagine the cause of the issue for beets is similar

**Comment by pimeys** [comment]
Yep, that's it. Fix in the PR.
